pkg/systemlogmonitor/log_monitor_test.go

```diff
@@ -151,26 +145,3 @@ func TestGenerateStatus(t *testing.T) {
 		}
 	}
 }
-
-func TestGoroutineLeak(t *testing.T) {
```

Comment:
Remember to drop this commit.

Reply:
Will keep that in mind.

Reply:
I think you still need to drop this commit, right? :)

Reply:
Done.